docs/builders/iso/Commented-Example.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: "docs"
+page_title: "XenServer-iso Commented Example"
+description: |-
+  This is a Commented example for how to use the xenserver-iso Packer Builder.
+  This Example is written in hcl
+---
+
+# What's this 
+This Example builds a vanilla Centos 8 Template. No Provisioning or Post-Processing will be done.
+
+If you want to run this example, you have to replace the following variables with your own values.
+```hcl
+  remote_host     = "xenserver.example.org"
+  remote_username = "root"
+  remote_password = "very-secret-password"
+  sr_iso_name     = "Local-ISO"
+  sr_name         = "Local-SR"
+  tools_iso_name  = "guest-tools.iso"
+```
+
+## Parts of this example
+This examples composes the following files from the examples folder
+
+* [centos8-example.pkr.hcl](../../examples/http/centos/centos8-example.pkr.hcl)
+* [ks-centos8-example.cfg](../../examples/http/centos/ks-centos8-example.cfg)
+
+
+## Explanation of centos8-example.pkr.hcl
```

Comment:
Same comment as the kickstarter file. I am worried that this will get out of sync with the hcl file. Since this is no longer JSON we can provide the comments in the HCL file and mention that the example is well commented.